src/patternfly/components/Table/table-tr.hbs

```diff
@@ -23,8 +23,8 @@
       {{ternary table-tr--IsHidden 'hidden' null}}
     {{/unless}}
   {{/if}}
-  {{ternary table--IsGrid 'role="row"'}}
-  {{ternary table-tr--IsClickable 'tabindex="0"'}}
+  {{ternary table--IsGrid 'role="row"' null}}
+  {{ternary table-tr--IsClickable 'tabindex="0"' null}}
```

Comment:
I'm seeing <tr class="pf-v6-c-table__tr" role&#x26;#x3d;&#x26;quot;row&#x26;quot;=""> so I think something is amiss here. (e.g. on http://localhost:5000/components/table/html/basic-table-example/)

Reply:
This issue was pre-existing and is now resolved 👍